src/app/header/header.component.html

```diff
@@ -1,4 +1,4 @@
-<div class="header">
+<div class="header header_main" [ngClass]="(isMainPage$ | async) ? 'header_main' : 'header_not_main'">
```

Comment:
please use meaningful class names

Reply:
class="header_main_page"
class="header_not_main_page"

Is it gonna be ok?

Reply:
@dmitryy90 
class="header_main_page"
class="header_not_main_page"

Is it gonna be ok?

Reply:
header > main  | header > secondary
or any descriptive class name
not_main makes no sense to me